app/scripts/directives/loading.dir.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+angular.module('yourStlCourts').directive('loading', function($timeout){
```

Comment:
Which is why I mentioned in the card that I had code for it and explained how it would look to John and Laura.